crates/nu-std/std/testing.nu

```diff
@@ -75,7 +55,7 @@ def get-annotated [
 # Annotations that allow multiple functions are of type list<string>
 # Other annotations are of type string
 # Result gets merged with the template record so that the output shape remains consistent regardless of the table content
-def create-test-record [] {
+def create-test-record [] nothing -> record<before-each: string, after-each: string, before-all: string, after-all: string, test: list<string>, test-skip: list<string>> {
```

Comment:
we really need to support writing things like
```nushell
def create-test-record [] nothing -> record<
    before-each: string,
    after-each: string,
    before-all: string,
    after-all: string,
    test: list<string>,
    test-skip: list<string>
> {
```
:eyes:

Reply:
Either that or custom types

Reply:
would love to see this :relieved: